<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>605</height>
   </rect>
  </property>
  <property name="icon" stdset="0" >
   <iconset resource="images.qrc" >:/images/galerist.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="2" column="1" >
     <widget class="GWidgets::PhotoControl" native="1" name="photoControl" />
    </item>
    <item row="0" column="0" colspan="3" >
     <widget class="GWidgets::PhotoView" name="imageList" />
    </item>
    <item row="1" column="1" >
     <widget class="GWidgets::SearchBar" native="1" name="searchBar" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionDefaultBackground" />
    <addaction name="actionRoundBackground" />
    <addaction name="separator" />
    <addaction name="actionShowProgress" />
    <addaction name="actionHideProgress" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionRemoveGallery" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="editMenu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuSelection" >
     <property name="title" >
      <string>&amp;Selection</string>
     </property>
     <addaction name="actionSelectAll" />
     <addaction name="actionDeselectAll" />
     <addaction name="actionInvertSelection" />
    </widget>
    <addaction name="actionAdd" />
    <addaction name="actionPreview" />
    <addaction name="actionRename" />
    <addaction name="actionDescribe" />
    <addaction name="actionRemove" />
    <addaction name="separator" />
    <addaction name="menuSelection" />
    <addaction name="separator" />
    <addaction name="actionConfiguration" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
    <addaction name="actionUpdate" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="editMenu" />
   <addaction name="viewMenu" />
   <addaction name="helpMenu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="galleryDock" >
   <property name="windowTitle" >
    <string>Gallery manager</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="GWidgets::GalleryTreeView" name="albumView" >
       <property name="editTriggers" >
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="separator" />
   <addaction name="actionAdd" />
   <addaction name="actionRemove" />
   <addaction name="separator" />
   <addaction name="actionDefaultBackground" />
   <addaction name="actionRoundBackground" />
  </widget>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/galerist.png</iconset>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quits the Galerist.</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Create gallery wizard</string>
   </property>
   <property name="iconText" >
    <string>Create gallery wizard</string>
   </property>
   <property name="toolTip" >
    <string>Create gallery wizard</string>
   </property>
   <property name="statusTip" >
    <string>Create a new gallery with the wizard.</string>
   </property>
  </action>
  <action name="actionDefaultBackground" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/default-rect.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Default background rectangle</string>
   </property>
   <property name="statusTip" >
    <string>Adds the default background rectangles.</string>
   </property>
  </action>
  <action name="actionRoundBackground" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/round-rect.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Round background renctangle</string>
   </property>
   <property name="statusTip" >
    <string>Adds the rounded background rectangles.</string>
   </property>
  </action>
  <action name="actionAdd" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/add.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Add new picture</string>
   </property>
   <property name="statusTip" >
    <string>Adds an individual picture to the selected gallery.</string>
   </property>
  </action>
  <action name="actionRemove" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/remove.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Remove selected pictures</string>
   </property>
   <property name="statusTip" >
    <string>Removes the selected pictures.</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionShowProgress" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/show-progress.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Show progress dialog</string>
   </property>
  </action>
  <action name="actionHideProgress" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/hide-progress.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Hide progress dialog</string>
   </property>
  </action>
  <action name="actionConfiguration" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configuration</string>
   </property>
  </action>
  <action name="actionPreview" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/zommselect.png</iconset>
   </property>
   <property name="text" >
    <string>Preview</string>
   </property>
  </action>
  <action name="actionRename" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/rename.png</iconset>
   </property>
   <property name="text" >
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDescribe" >
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/describe.png</iconset>
   </property>
   <property name="text" >
    <string>Describe</string>
   </property>
  </action>
  <action name="actionSelectAll" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Select all</string>
   </property>
  </action>
  <action name="actionDeselectAll" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Deselect all</string>
   </property>
  </action>
  <action name="actionInvertSelection" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Invert selection</string>
   </property>
  </action>
  <action name="actionRemoveGallery" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/remove.png</iconset>
   </property>
   <property name="text" >
    <string>Remove gallery</string>
   </property>
  </action>
  <action name="actionUpdate" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >:/images/upgrade.png</iconset>
   </property>
   <property name="text" >
    <string>Check for &amp;updates</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GWidgets::PhotoView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/photoview.h</header>
  </customwidget>
  <customwidget>
   <class>GWidgets::PhotoControl</class>
   <extends>QWidget</extends>
   <header>widgets/photocontrol.h</header>
  </customwidget>
  <customwidget>
   <class>GWidgets::GalleryTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/gallerytreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GWidgets::SearchBar</class>
   <extends>QWidget</extends>
   <header>widgets/searchbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
